demoproductform/demoproductform.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * 2007-2020 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0).
+ * It is also available through the world-wide-web at this URL: https://opensource.org/licenses/AFL-3.0
+ */
+
+declare(strict_types=1);
+
+use PrestaShop\Module\DemoProductForm\Form\Modifier\ProductFormModifier;
+use PrestaShop\Module\DemoProductForm\Install\Installer;
+use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
+use Symfony\Component\Translation\TranslatorInterface;
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+class DemoProductForm extends Module
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    public function __construct()
+    {
+        $this->name = 'demoproductform';
+        $this->author = 'PrestaShop';
+        $this->version = '1.0.0';
+        $this->ps_versions_compliancy = ['min' => '1.7.8.0', 'max' => _PS_VERSION_];
```

Comment:
Isn't it dangerous to gamble on future this way ? I mean if Prestashop update symfony to v6, will the module still work ?
Why not trying to use composer-like description ?

Reply:
good questions.
I actually don't know best practices in this topic. (I just copied these from other example modules and focused on the code :shushing_face: :smile: ) Would appreciate if someone else answers those :v:

Reply:
What do you think @PrestaShop/prestashop-core-developers ? ^^

Reply:
What do you mean @MeKeyCool about:
>Why not trying to use composer-like description ?

Setting `'max' => _PS_VERSION_` allow us to not release a new version of the module each time a new PrestaShop version is released and we can release a new version of the module either to change the `'max' => _PS_VERSION_` or to make it compatible with the latest PrestaShop version. It's not ideal but I think that's the best we can do ATM.